CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+Change Log
+====================
+
+Introduction
+------------
+Drake is undergoing continuous development and some of the changes may break its API and ABI. To help downstream users cope, this document provides brief summaries of the changes. The changes are organized by pull requests that are merged into Drake's master branch.
```

Comment:
Agree with @jwnimmer-tri.  Proposed alternative wording: "Drake is undergoing continuous development and cannot yet offer a stable, documented API or ABI."